app/src/main/java/ch/epfl/cs311/wanderwave/model/spotify/SpotifyController.kt

```diff
@@ -305,6 +307,87 @@ constructor(
     }
   }
 
+  /**
+   * Retrieve the image URL of an album.
+   *
+   * @param albumId The Spotify ID of the album.
+   * @return A Flow that emits the album image URL as a string.
+   */
+  fun getAlbumImage(albumId: String): Flow<String?> {
+    return callbackFlow {
+      val accessToken = authenticationController.getAccessToken()
```

Comment:
why are you putting the accessToken here, isn't it check in the authenicationController?

Reply:
I removed it by calling spotifyGetFromURL which already do it